### DISCUSSION TITLE
Cookie介紹

### DISCUSSION THREAD

**Discussion by minniewei** [1101-GS4719-C/Python-Community-GS4719-C#541] [General]
Cookie在封包中扮演很重要的角色，透過這篇文章我終於知道cookie具體的原理了!!!希望分享給和我之前一樣覺得cookie很深奧的朋朋們!!!(之後再來找找看session的相關文章)
https://progressbar.tw/posts/91

**Comment by JSHT** [comment]
![image](https://user-images.githubusercontent.com/48956859/161470656-5a2e3168-ac23-4a14-a55b-59581534c86a.png)
圖片來源：[Computer Networking: A Top-down Approach](https://view.officeapps.live.com/op/view.aspx?src=https%3A%2F%2Fgaia.cs.umass.edu%2Fkurose_ross%2Fppt-8e%2FChapter_2_v8.1.pptx&wdOrigin=BROWSELINK) 第 34 張

**Comment by ThomasMeng0515** [comment]
要了解網頁的運作，cookie算是一個重要的概念，謝謝分享

**Comment by hapiwangy** [comment]
Cookie的概念在網路裡面十分常用到，謝謝同學的介紹。

**Comment by SamTU0222** [comment]
謝謝分享 原來cookie是這樣的概念

**Comment by lx3211** [comment]
原來我們勾選的記住帳號密碼就是cookie的開關呀

**Comment by HuanYuuuuuu** [comment]
cookies 會紀錄用戶的資訊，像是購物車、會員登入帳號密碼或瀏覽紀錄、停留時間。
是一個非常方便的功能

**Comment by Ku-Hao-Chen** [comment]
謝謝分享 讓我知道cookie是甚麼。

**Comment by sean1325418** [comment]
終於知道一些網頁要允許的cookie是甚麼東西了

**Comment by Sazmafuji** [comment]
cookie原來是這樣，每次用網頁時常跑出cookie..都不知在做啥，現在有微了解在做甚麼了

**Comment by henrywen547** [comment]
之前我對於cookie是甚麼並沒有完全理解，這篇文章用日常生活去解說，讓我終於了解cookie是甚麼以及背後如何運作了

**Comment by bloodtrail0511** [comment]
原來是這樣，不然我看到都不知道是甚麼，只知道案接受

**Comment by Lai27330** [comment]
常常打開網頁時就會出現cookie的請求，每次都覺得資安好像會受到威脅，但沒想到儲存密碼這個方便的設計本身就是cookie的設計，原來在不知不覺中早就允許過這個功能了，看來以後按下各種按鈕都要更加謹慎了，謝謝同學對 coookie的分享。

**Comment by yi-1111** [comment]
終於明白這背後的概念了，原來是醬子阿，謝謝分享

**Comment by Hchejxnc** [comment]
原來cookie的概念是這樣~

**Comment by Yuhsin000** [comment]
原來在不知不覺中就已經有使用cookie的功能了，以後不儲存帳密了，保護資料!

**Comment by d26488399** [comment]
感謝分享，終於知道cookie是甚麼了

**Comment by awkward-willy** [comment]
謝謝分享，讓我對 cookie 有更深入的理解
不過 Google 希望能在 2023 年讓 Chrome 停用第三方 cookie
因為已延過一年了，不知道未來會不會真的實現

**Comment by ghost** [comment]
cookie感覺是真的有那麼一回事@@

**Comment by Rosancl** [comment]
感謝分享，文中舉的例子讓cookie的概念簡單易懂！

**Comment by stan890425** [comment]
謝謝分享，以後就知道拒絕cookie是甚麼了~~

**Comment by jcpan821** [comment]
原來我在不知不覺中用了cookie

**Comment by leon10456** [comment]
感謝樓主分享

**Comment by liangyinglly** [comment]
原來cookie是屬於一種小型的文字檔案，透過加密的方式儲存在用戶端上的資料，以前什麼都不知道就只會按同意

**Comment by etin2468** [comment]
我之前就只會按同意而已，原來Cookie的概念在網路裡面十分常用到，謝謝同學的介紹。

**Comment by Lynn12378** [comment]
原來cookies是這樣子的，感謝分享!

**Comment by 110409538** [comment]
感謝分享！！

**Comment by Jimmy911218** [comment]
謝謝分享，解決了我對他的困惑
